net-misc/cadaver/cadaver-0.23.3.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="A command line WebDAV client"
+DESCRIPTION="It supports file upload, download, on-screen display, namespace ope
+rations (move/copy), collection creation and deletion, and locking operations."
+HOMEPAGE="http://www.webdav.org/cadaver"
+COPYRIGHT="1998-2008 Joe Orton,
+   except where otherwise indicated.
+   2002-2003 GRASE Lab, UCSC."
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://www.webdav.org/cadaver/cadaver-0.23.3.tar.gz"
+CHECKSUM_SHA256="fd4ce68a3230ba459a92bcb747fc6afa91e46d803c1d5ffe964b661793c13fca"
+#PATCHES=""
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	cadaver=$portVersion
+	cmd:cadaver
+	"
+REQUIRES="
+	haiku
+	lib:libintl
+	lib:libexpat
+	lib:libxml2
+	lib:libssl
+	lib:libreadline
+	lib:libncurses
+	lib:libzlib
+	lib:libiconv
+
```

Comment:
```diff
 REQUIRES="
 	haiku
-	lib:libintl
 	lib:libexpat
+	lib:libiconv
+	lib:libintl
-	lib:libxml2
-	lib:libssl
+	lib:libncurses
+	lib:libreadline
 	lib:libssl
+	lib:libxml2
-	lib:libncurses
 	lib:libzlib
-	lib:libiconv
-
-        "
+	"
```
(That's just sorting. BTW, if you copy/paste please make sure you have tabs, not spaces.)